engine/crates/parser-openapi/src/tests/snapshots/parser_openapi__tests__petstore_output-2.snap

```diff
@@ -2166,12 +2166,14 @@ Registry {
         "PetstoreUser": FederationEntity {
             keys: [
                 FederationKey {
-                    selections: [
-                        Selection {
-                            field: "username",
-                            selections: [],
-                        },
-                    ],
+                    selections: FieldSet(
+                        [
+                            Selection {
+                                field: "username",
+                                selections: [],
+                            },
+                        ],
+                    ),
```

Comment:
Is the registry the data structure we serialize and retrieve from KV that we have to keep backwards compatible? Federation support is very new so breaking changes would be ok here, just checking my understanding.

Reply:
Ah yes, I misread that, I thought something became an array that was not, but it's just indentation that looks like that. Yep the newtype should be transparent.